content/en/references/network-troubleshooting/created-resources/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Accessing a resource created by LocalStack
+weight: 3
+tags:
+- troubleshooting
+- networking
+---
+
+You are trying to access a resource that was created by LocalStack, for example an OpenSearch cluster or RDS database.
+
+When created, these resources are accessible from a URL, or they will return a hostname that use to access the resource.
+In general, the environment variable `HOSTNAME_EXTERNAL` can be used to control what hostname is returned, though currently not in every case.
+For more details about specific situations, see below.
+
+{{<alert title="Note">}}
+An additional environment variable is available: `LOCALSTACK_HOSTNAME`.
+This is primarily used for communication within LocalStack and should not be used.
```

Comment:
This part might be a bit confusing for users since `LOCALSTACK_HOSTNAME` is actually the primary one they'll usually use in Lambda functions :thinking:

Reply:
Thanks for mentioning this - yeah I am certainly confused by `HOSTNAME_EXTERNAL` and `LOCALSTACK_HOSTNAME`. I can only imagine what it's like for our users! I want to get rid of this duality and confusion, but for now I'll try to add clarity - thanks!